### PR TITLE
improvement: log namespace and cloud URL

### PR DESCRIPTION
**Pull request by eysi09**
**What this PR does / why we need it**:

Prints the namespace Garden is running in in the header and the URL to the project in Garden Cloud if applicable.

Screenshot 2021-09-13 at 09 20 36

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: core/src/cli/cli.ts [hunk 1]
```diff
@@ -236,13 +237,18 @@ ${renderCommands(commands)}
 
     contextOpts.persistent = persistent
     const { streamEvents, streamLogEntries } = command
+    // Print header log before we know the namespace to prevent content from
+    // jumping.
+    // TODO: Link to Cloud namespace page here.
+    const nsLog = headerLog.info(chalk.white(`Running in namespace ...\n`))
```
Comment (thsig) [inline core/src/cli/cli.ts:243]: Good stuff!

I think a darker / gray colour "Running in namespace" would be cleaner (while keeping the white for the actual namespace name).

Comment (eysi09) [reply core/src/cli/cli.ts:243]: Like this?

Comment (thsig) [reply core/src/cli/cli.ts:243]: Yeah, that's good! I think I'd prefer an even darker gray (like in "Authorizing..."), but either works for me.

Comment (eysi09) [reply core/src/cli/cli.ts:243]: It's the same color, `chalk.gray`.

**Review by thsig — APPROVED** [review]

**Review by thsig — APPROVED** [review]

**Pull request merged**